src/static/css/index.css

```diff
@@ -68,14 +68,14 @@ a{
     font-family: 'Oswald', sans-serif;
 }
 #seconds , #minutes {
-    font-size: 5em;
+    font-size: 2em;
```

Comment:
@Amulya-coder Have some padding too.

Reply:
@vinitshahdeo  Yes adding padding to it looks good.

Reply:
@vinitshahdeo Now Is It OK

![TimerPic](https://user-images.githubusercontent.com/66437295/92088798-fc585900-edea-11ea-8a6e-45d3393b6d1b.png)

Reply:
Yes, thanks 👍